error.go

```diff
@@ -7,6 +7,20 @@ import (
 	"net/http"
 )
 
+const (
+	// Access denied
+	UnauthorizedCode = 999
```

Comment:
This is not an original response code but i think its good to have this to handle errors of UIPath response.